### ISSUE TITLE
boinc 5.10.45 wont suspend when battery linux

### ISSUE THREAD

**Issue by romw** [BOINC/boinc-ng#657]
**Reported by stompdagger1 on 3 Nov 38452191 11:06 UTC**
this issue has worked in version 5.10.28, now it seems that when I unplug the AC adapter, boinc still continue to calculate even when the pref file says not to.
running 64bit gentoo kernel 2.6.24 and boinc 5.10.45

more details at http://boinc.berkeley.edu/dev/forum_thread.php?id=2775

**Comment by romw** [comment]
**Commented by Nicolas on 8 Sep 38452688 08:53 UTC**
Interesting; did that '''ever''' work before?

**Comment by romw** [comment]
**Commented by stompdagger1@yahoo.com on 18 Oct 38454181 14:13 UTC**
Replying to [Nicolas](comment:1):
> Interesting; did that '''ever''' work before?
for me yes, worked like a charm

**Comment by romw** [comment]
**Commented by mjakubicek on 3 Aug 38463700 06:13 UTC**
Hm, I have no problems with a 64 bit laptop running Fedora 9 with kernel 2.6.25.6.

a) Could you boot a Live CD/DVD of Gentoo (same version) on another laptop, install BOINC and try if it works?

b) Could you boot a Live CD/DVD of Fedora 9 on your laptop, install BOINC and try if it works?

It could be just some kernel/ACPI weirdness regarding the HW (a regression, actually, if it worked before) or something Gentoo specific, IMO.

Does it work with your current kernel and BOINC 5.10.28?
If not, does it work with some previous kernel and current BOINC (5.10.45)?

**Comment by romw** [comment]
**Commented by mjakubicek on 7 Mar 38465542 13:46 UTC**
...just another idea: does HAL detect AC adapter correctly?

Type: 

```
hal-get-property --udi `hal-find-by-capability --capability ac_adapter` --key 'ac_adapter.present'
```


It should return true/false if the adapter is/is not present.

**Comment by romw** [comment]
**Commented by stompdagger1@yahoo.com on 30 Jul 38466401 11:06 UTC**
already worked with david and figured it out, the current way boinc is watching for the acpi is using /proc, /proc is deprecated, so it will probably be dropped from newer kernels.
but it can be done using the /sys, I've passed all the info to david.

**Comment by romw** [comment]
**Commented by stompdagger1@yahoo.com on 5 Sep 38487363 11:06 UTC**
dave, any update on this issue for the next version?

**Comment by romw** [comment]
**Commented by davea on 17 Feb 38488335 06:40 UTC**
Possibly fixed in [15492]

**Comment by romw** [comment]
**Commented by stompdagger1@yahoo.com on 9 Jun 38619178 07:33 UTC**
hello again, I've just wanted to know if the fix is included in version 6.2.16?

**Comment by romw** [comment]
**Commented by davea on 20 May 38619181 16:53 UTC**
I think it's fixed in 6.2.16, but I haven't tested it.

**Comment by romw** [comment]
**Commented by stompdagger1@yahoo.com on 15 May 38620952 23:33 UTC**
I run 6.2.16, it doesn't seems to work, when I unplug it, the manager stills continue to run
